### DISCUSSION TITLE
[1150530] Wizard with a Gun 荒野枪巫

### DISCUSSION THREAD

**Discussion by BitAstray** [pjy612/SteamManifestCache#39] [Share]
appid:1150530
keys:
1150531;be7f839c9405122842ab1a51bb4f328f4d7a3e716a4dd212656b6d59625f6175
[1150531_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13685098/1150531_config.vdf.txt)

email: bitcreate@qq.com

**Comment by BitAstray** [comment]
code:B969606DA7E4F7F963682475EFB20B61BE858D2F

**Reply by pjy612** [reply]
请查收邮件
